dev/setup/codesniffer/ruleset.xml

```diff
@@ -17,8 +18,7 @@
 	<exclude-pattern type="relative">*/nltechno*</exclude-pattern>
 	<exclude-pattern type="relative">source</exclude-pattern>
 	<exclude-pattern type="relative">.git</exclude-pattern>
-	<exclude-pattern>htdocs/includes</exclude-pattern>
```

Comment:
This directory contains code that we do not control. They are external sources.
So if we apoly the dolibarr rule checks on it, we should get a lot of errors, shouldn't we ?

Reply:
I agree about the directory but it's already excluded in a rule above.